oak/server/grpc_stream.h

```diff
@@ -35,19 +35,41 @@ class GrpcStream {
   GrpcStream(std::shared_ptr<OakNode> node_)
       : server_reader_writer_(&server_context_), node_(node_) {}
 
+  void HandleCreateStream() {
+    LOG(INFO) << "gRPC Event: New stream created";
+    auto callback = new std::function<void()>([this]() { HandleReadEvent(); });
```

Comment:
It might make sense to typedef the function type to give it a name and ensure that the right type is used